.gitignore

```diff
@@ -33,3 +33,7 @@ velocity.log
 .project
 .settings
 .vscode
+/tests/ballerina-test/desc_gen/
+/tests/ballerina-test/google/
+/composer/modules/web/package.json
```

Comment:
@Shan why are we ignoring package.json?

Reply:
@hevayo - Local builds modify these files and it is causing troubles (with applying stashes, etc) when working with multiple branches. Should I remove these entries?

Reply:
What does get changed in package.json ? If it is the version number the proper fix is to update the release build to commit the change.